tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -311,12 +311,6 @@ internal static partial class ConfigurationKeys
         /// <seealso cref="TracerSettings.IpHeader"/>
         public const string IpHeader = "DD_TRACE_CLIENT_IP_HEADER";
 
-        /// <summary>
-        /// Configuration key indicating if the header should not be collected. The default for DD_TRACE_CLIENT_IP_HEADER_DISABLED is false.
-        /// </summary>
-        /// <seealso cref="TracerSettings.IpHeaderDisabled"/>
-        public const string IpHeaderDisabled = "DD_TRACE_CLIENT_IP_HEADER_DISABLED";
-
```

Comment:
I wonder if this is correct? It effectively means that customers _can't_ disable IP header collection, even if they have security enabled? e.g. should the conditions essentially be

```csharp
if(Security.Instance.Settings.Enabled && !tracer.Settings.IpHeaderDisabled)
{
    //
```

Reply:
Yes, in the end, it's been decided that ASM enabled implies always sending the ip, regardless of any apm side configuration, as ASM without ip collecting doesn't make much sense, as blocking per ip is one of the main feature.